src/plumpy/rmq/communications.py

```diff
@@ -96,15 +73,16 @@ def converted(communicator: kiwipy.Communicator, *args: Any, **kwargs: Any) -> k
             return kiwi_future
 
         msg_fn = functools.partial(coro, communicator, *args, **kwargs)
-        task_future = futures.create_task(msg_fn, loop)
-        return plum_to_kiwi_future(task_future)
+        task_future = create_task(msg_fn, loop)
+        return wrap_to_concurrent_future(task_future)
 
     return converted
 
 
-def wrap_communicator(
-    communicator: kiwipy.Communicator, loop: Optional[asyncio.AbstractEventLoop] = None
-) -> 'LoopCommunicator':
+T = TypeVar('T', bound=kiwipy.Communicator)
```

Comment:
Can we give more meaningful name?
```suggestion
SubtypeCommunicator = TypeVar('SubtypeCommunicator', bound=kiwipy.Communicator)
```
or
```suggestion
CommunicatorT = TypeVar('CommunicatorT', bound=kiwipy.Communicator)
```
or anything else that references that it comes from `Communicator`

Reply:
Would be better, I'll change.

Reply:
https://github.com/aiidateam/plumpy/pull/297/commits/00d5be962504b19f838e9202d98f281dbe59e078